src/general/EnvironmentalTolerances.cs

```diff
@@ -8,31 +9,42 @@ public class EnvironmentalTolerances
     /// <summary>
     ///   Temperature (in C) that this species likes to be in
     /// </summary>
-    public float PreferredTemperature = 15;
+    public float PreferredTemperature;
 
     /// <summary>
     ///   How wide a temperature range this species can stay in effectively. The range of temperatures is
     ///   <c>PreferredTemperature - TemperatureTolerance</c> to <c>PreferredTemperature + TemperatureTolerance</c>
```

Comment:
Now it is only PreferredTemperature + TemperatureTolerance, right?

Reply:
Temperature has not been changed, it is still Preferred +/- Tolerance, this is indicated in the UI by the line connecting the range to the grabber being drawn from the center of the range as opposed to pressure where it is drawn from the minimum.